eng/scripts/Set-SkipJobVariables.ps1

```diff
@@ -0,0 +1,60 @@
+. "$PSScriptRoot/../common/scripts/common.ps1"
```

Comment:
This should be integrated into New-BuildInfo.ps1.  That script is responsible for forming the Build, Live Test and Smoke Test matrices.  If we decide that there's no build or test matrix for a PR, it should be reflected there.